privacy_evaluator/models/tf/cnn.py

```diff
@@ -30,7 +32,7 @@ def ConvNet(
 
 class ConvNetMNIST(keras.Model):
     def __init__(
-        self, num_classes: int = 2, num_channels: Tuple[int, ...] = (1, 64, 128)
+        self, num_classes: int = 2, num_channels: Tuple[int, ...] = (1, 16, 32, 64)
     ):
```

Comment:
Same question as above, is there a specific reason for these default value of num_classes?